public/app/features/alerting/unified/components/rule-editor/NotificationsStep.tsx

```diff
@@ -1,112 +1,263 @@
+import { css } from '@emotion/css';
 import React from 'react';
 import { useFormContext } from 'react-hook-form';
 
-import { Icon, Text, Stack } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { Icon, RadioButtonGroup, Stack, Text, useStyles2 } from '@grafana/ui';
 
 import { RuleFormType, RuleFormValues } from '../../types/rule-form';
 import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
 
 import LabelsField from './LabelsField';
 import { NeedHelpInfo } from './NeedHelpInfo';
 import { RuleEditorSection } from './RuleEditorSection';
+import { SimplifiedRouting } from './alert-rule-form/simplifiedRouting/SimplifiedRouting';
 import { NotificationPreview } from './notificaton-preview/NotificationPreview';
 
 type NotificationsStepProps = {
   alertUid?: string;
 };
+
+enum RoutingOptions {
+  'notification policy' = 'notification policy',
+  'contact point' = 'contact point',
+}
+
 export const NotificationsStep = ({ alertUid }: NotificationsStepProps) => {
-  const { watch } = useFormContext<RuleFormValues & { location?: string }>();
+  const { watch, setValue } = useFormContext<RuleFormValues & { location?: string }>();
+  const styles = useStyles2(getStyles);
 
-  const [type, labels, queries, condition, folder, alertName] = watch([
+  const [type, labels, queries, condition, folder, alertName, manualRouting] = watch([
     'type',
     'labels',
     'queries',
     'condition',
     'folder',
     'name',
```

Comment:
Determines the routing option.
I decided to keep both `contactPoints` and `manualRouting` fields in the form to be able to restore the value of the selected contact points while switching the routing option.
However, if this second tab(using policies) is selected when saving, these contact points will be removed.(not implemented yet)